source/contribute/guidelines.md

```diff
@@ -59,14 +59,14 @@ Due to a `pip` limitation, it is required to increment the major version number
 
 #### Peer reviews
 
-Pull requests should generally be **reviewed** by someone else than their authors.
+Pull requests should generally be **reviewed** by someone other than their authors.
 
 This is mandatory for:
 
 - Any Pull Request with **breaking changes** on `openfisca-france`, `openfisca-web-api`.
 - Any Pull Request bringing **new features**, if these features are not relative to a specific scope.
   + Adding a new route to the API **requires** a review.
-  + A review is yet not mandatory to add a new formula to social contributions in `openfisca-france`. It is though recommended.
+  + A review is not yet mandatory to add a new formula to social contributions in `openfisca-france`. It is though recommended.
```

Comment:
This is all outdated and will need a more thorough update, but let's either drop this part or not update it.
```suggestion
```